app/controllers/welcome_controller.rb

```diff
@@ -28,6 +29,20 @@ def mentor
     @report = params[:report].blank? ? "basics" : ERB::Util.html_escape(params[:report])
   end
 
+  def participant
+    @participant = @current_user.person
+    @participant.type = "Participant"
+    @participant.save
```

Comment:
Not sure what's happening here... it looks like if anyone navigates to this page, they would magically become a Participant regardless of their person type. Seems like a bad idea?